### PR TITLE
DB To Official Postgresql

### PR DESCRIPTION
**Pull request by gpmidi**

### CODE REVIEW

File: musicbrainz-postgres/Dockerfile [hunk 1]
```diff
@@ -1,33 +1,23 @@
...
+
+# Deps
+RUN echo "deb http://http.us.debian.org/debian stable main contrib non-free" > /etc/apt/sources.list.d/nonfree.list
+RUN apt-get update
+RUN apt-get install -y adduser sudo git unzip unrar \
+        bash-completion wget vim build-essential \
+        postgresql-server-dev-9.4 \
+        postgresql-plperl-9.4 libjson-xs-perl libicu-dev \
+ && apt-get clean autoclean \
+ && apt-get autoremove -y
```
Comment (rickatnight11) [inline musicbrainz-postgres/Dockerfile]: Will you re-add the apt cleanup?

```
rm -rf /var/lib/{apt,dpkg,cache,log}/ && \
```

**Comment by rickatnight11** [comment]
Why remove the DB initialization scripts?

EDIT: Ah, I see, you're using the postgres image's environment variable support to set defaults.  I like it.  I'll need to modify this a bit to support the default variables in the `DBDefs.pm` example.

File: musicbrainz-postgres/Dockerfile (hunk 1 above)
Comment (gpmidi) [inline musicbrainz-postgres/Dockerfile]: Sure. I'll do it in the morning.

**Comment by isuftin** [comment]
:clap: This is great! I'm glad this is getting traction again.

File: musicbrainz-postgres/Dockerfile [hunk 2]
```diff
@@ -1,33 +1,23 @@
...
+ENV POSTGRES_PASSWORD=musicbrainz
+ENV POSTGRES_DB=initial
+
+# Deps
+RUN echo "deb http://http.us.debian.org/debian stable main contrib non-free" > /etc/apt/sources.list.d/nonfree.list
+RUN apt-get update
+RUN apt-get install -y adduser sudo git unzip unrar \
+        bash-completion wget vim build-essential \
+        postgresql-server-dev-9.4 \
+        postgresql-plperl-9.4 libjson-xs-perl libicu-dev \
```
Comment (isuftin) [inline musicbrainz-postgres/Dockerfile:15]: What's the purpose of adding bash-completion, vim, adduser and sudo here?

Comment (gpmidi) [inline musicbrainz-postgres/Dockerfile:15]: I frequently need to enter an instance during troubleshooting. Having those tools in place makes that a lot easier. Plus they don't take up much space.

**Comment by gpmidi** [comment]
Added requested cleanup line

**Pull request merged**